qa/tasks/cephfs/test_admin.py

```diff
@@ -760,13 +784,44 @@ def test_fs_rename_fails_with_mirroring_enabled(self):
         new_fs_name = 'new_cephfs'
 
         self.run_ceph_cmd(f'fs mirror enable {orig_fs_name}')
+        self.run_ceph_cmd(f'fs fail {self.fs.name}')
+        sleep(5)
         try:
             self.run_ceph_cmd(f'fs rename {orig_fs_name} {new_fs_name} --yes-i-really-mean-it')
         except CommandFailedError as ce:
             self.assertEqual(ce.exitstatus, errno.EPERM, "invalid error code on renaming a mirrored file system")
         else:
             self.fail("expected renaming of a mirrored file system to fail")
         self.run_ceph_cmd(f'fs mirror disable {orig_fs_name}')
+        self.run_ceph_cmd(f'fs set {self.fs.name} joinable true')
+        sleep(5)
+
+    def test_renaming_when_fs_is_not_down(self):
+        '''
+        Test that the command "ceph fs swap" command fails when first of the
+        two of FSs isn't failed/down.
+        '''
+        client_id = 'test_new_cephfs'
+        new_fs_name = 'new_cephfs'
+        msg = ('All MDSs for an FS must be down before renaming the file '
+               'system. See `ceph fs fail`.')
+        self.negtest_ceph_cmd(
+            args=(f'fs rename {self.fs.name} {new_fs_name} --yes_i_really_mean_it'),
+            retval=errno.EPERM, errmsgs=msg)
```

Comment:
@rishabh-d-dave Do we still need error string checks?

Reply:
Removed.